### ISSUE TITLE
Update to Docker 1.7.0

### ISSUE THREAD

**Issue by hadronzoo** [Parallels/boot2docker-vagrant-box#11]
[Docker 1.7.0 has been released](https://github.com/docker/docker/blob/master/CHANGELOG.md#170-2015-06-16) and the 1.7 client is incompatible with a 1.6 server.

**Comment by caseywebdev** [comment]
:+1:

**Comment by yurtaev** [comment]
:+1:

**Comment by zfogg** [comment]
My team works with Docker regularly so, I keep an up-to-date fork of this project :)

https://github.com/wearableintelligence/boot2docker-vagrant-box/releases/tag/docker%2Fv1.7.0

(sorry to @Parallels for the cheeky Github release naming scheme :P heh)

**Comment by legal90** [comment]
Guys, sorry for so long delay, I had to spend some time to renew our scripts and template.
I've just prepared a **pre-release version** of parallels/boot2docker box v1.7.0 with some significant changes:
https://github.com/Parallels/boot2docker-vagrant-box/releases/tag/parallels%2Fv1.7.0

Usage information was also updated: https://github.com/Parallels/boot2docker-vagrant-box#usage

@hadronzoo @yurtaev @caseywebdev @zfogg Could you please verify it on your cases? 
Since I don't use Docker regularly, I'd like to get a feedback about these changes:

> TLS is enabled by default. Certificates are syncing automatically to the ./tls/ directory on every vagrant up.

1) Is it OK that TLS certs is always syncing automatically? It is implemented via shell provision in the box's  Vagrantfile (https://github.com/Parallels/boot2docker-vagrant-box/blob/parallels/vagrantfile.tpl#L32-L33)

2) Usage info contains this advice for setting environment variables:
```bash
export DOCKER_CERT_PATH=`pwd`/tls
export DOCKER_HOST="tcp://`vagrant ssh-config | sed -n "s/[ ]*HostName[ ]*//gp"`:2376"
```
Obviously, these commands should be executed in the same directory where `vagrant up` was executed. Is it acceptable to your use case? Or do you use some other ways to fetch these values?

**Comment by legal90** [comment]
It could be tested with:
```bash
vagrant box add parallels-test-b2d https://github.com/Parallels/boot2docker-vagrant-box/releases/download/parallels%2Fv1.7.0/boot2docker-parallels.box
vagrant init parallels-test-b2d
vagrant up
```

**Comment by hadronzoo** [comment]
Thank you for the update. Is there a reason to enable TLS for this use case? It doesn't seem worth the added complexity and overhead IMO.

**Comment by legal90** [comment]
> Thank you for the update. Is there a reason to enable TLS for this use case? It doesn't seem worth the added complexity and overhead IMO.

@hadronzoo Yes, it looks complicated, but TLS is enabled by default on the server side of boot2docker starting since version 1.3.0. So, most likely users expect that Docker server is running with TLS and it is listening the port 2376.

However, seems like automatic certificate syncing to `./tls/` trough the "shell" provisioner is not a good idea. This approach will not work in case of multi-machine configuration in single Vagrantfile. I think it will be better to remove this provision step and add instruction to to README.md about manual certificate syncing.

Also, I think it will be useful to mention in README.md tip how to disable TLS on the server side. It's quite simple:
```
vagrant ssh -c "sudo sh -c 'echo \"export DOCKER_TLS=no\" > /var/lib/boot2docker/profile'"
vagrant reload
```
After that it will be possible to work with unsecure connection:
```
export DOCKER_TLS_VERIFY="0"
export DOCKER_HOST="tcp://`vagrant ssh-config | sed -n "s/[ ]*HostName[ ]*//gp"`:2375" 
```
Any suggestions? How many people are really need for TLS for Docker on their vagrant machines? Wondering, how do they sync certificates now.

**Comment by legal90** [comment]
_"Better late than never"_. Finally, it's released now: [v1.7.0](https://github.com/Parallels/boot2docker-vagrant-box/releases/tag/parallels%2Fv1.7.0) 
And published on Atlas as well: https://atlas.hashicorp.com/parallels/boxes/boot2docker/versions/1.7.0

The upcoming v1.7.1 version will be released much faster :)
